src/components/store/Dapp.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="tw-border tw-m-4 tw-w-72 tw-rounded-lg">
+    <div class="tw-flex tw-flex-grow tw-cursor-pointer tw-p-4" @click="emitClickEvent">
+      <Avatar :url="dapp.iconUrl" class="tw-w-14 tw-h-14" />
+      <div class="tw-ml-4">
+        <div class="tw-text-lg tw-font-semibold">{{ dapp.name }}</div>
```

Comment:
![Screen Shot 2021-10-03 at 1 36 14 PM](https://user-images.githubusercontent.com/40356749/135751945-6eb2090b-db10-4b02-92ae-2960d3b8baaf.png)
The font color do not change in dark mode